rust_snuba/src/consumer.rs

```diff
@@ -80,6 +80,8 @@ pub fn consumer_impl(
 
     // TODO: Support multiple storages
     assert_eq!(consumer_config.storages.len(), 1);
+
+    // TODO: Remove this once errors is implemented in Rust
```

Comment:
can't we remove it now?

Reply:
Since errors is not actually working properly yet, I figured it's best to leave it in